src/blocks/card.css

```diff
@@ -0,0 +1,89 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background-color: var(--white-color);
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px ;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;

```

Comment:
checklist p.9 [STYLES] - Don't hardcode exact height size. Add necessary paddings according to mockup and let content dictate the container size.

Reply:
I did not set the exact hight for whole container only for image , if img hight set auto -> high become 132.2px and tests are not passed